client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "corona-tracker",
-  "version": "0.2.28",
+  "version": "0.3.0",
```

Comment:
Sorry I may have worded it poorly, we're at 0.3.0 now need to go to 0.3.1

Reply:
Done